### DISCUSSION TITLE
Total commits don't count archived repo commits count

### DISCUSSION THREAD

**Discussion by sametcelikbicak** [anuraghazra/github-readme-stats#2250] [Q&A] [answered]
### Describe the bug

Hi all, I'm using below stats and it does not count public archive repo commits count. My total commit greater then `1k` but now it look like `~23`

my config: `https://github-readme-stats.vercel.app/api?username=sametcelikbicak&show_icons=true&theme=algolia&include_all_commits=true&count_private=true`

  <img height="180em" src="https://github-readme-stats.vercel.app/api?username=sametcelikbicak&show_icons=true&theme=algolia&include_all_commits=true&count_private=true"/>

If I remove `count_private=true` count was changed.
`https://github-readme-stats.vercel.app/api?username=sametcelikbicak&show_icons=true&theme=algolia&include_all_commits=true`
  <img height="180em" src="https://github-readme-stats.vercel.app/api?username=sametcelikbicak&show_icons=true&theme=algolia&include_all_commits=true"/>

### Expected behaviour

It would be great stats count then public archive repositories commits.

**Comment by rickstaa** [comment, marked as answer]
@sametcelikbicak Thanks for your question. Your cards look fine now. I think this was because of a caching error (see https://github.com/anuraghazra/github-readme-stats#common-options). I created https://github.com/anuraghazra/github-readme-stats/pull/2177 to fix incorrect results from being cached.

**Reply by sametcelikbicak** [reply]
@rickstaa Thank for information, I checked my car is look fine now. 👍
